src/System.Net.Security/tests/UnitTests/System.Net.Security.Unit.Tests.csproj

```diff
@@ -57,4 +57,13 @@
     </Compile>
   </ItemGroup>
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.targets))\dir.targets" />
+  <Target Name="RemoveProductionReference" BeforeTargets="CoreCompile">
```

Comment:
This should be explained in a comment here

---
In reply to: [81845354](https://github.com/dotnet/corefx/pull/12351#discussion_r81845354) [](ancestors = 81845354)